runners/direct-java/src/main/java/org/apache/beam/runners/direct/TransformEvaluatorRegistry.java

```diff
@@ -132,9 +131,6 @@ public static class DirectTransformsRegistrar implements TransformPayloadTransla
           .put(
               DirectTestStream.class,
               TransformPayloadTranslator.NotSerializable.forUrn(DIRECT_TEST_STREAM_URN))
-          .put(
```

Comment:
If I understood correctly, this is because we now have an AutoService in runners/core-java/src/main/java/org/apache/beam/runners/core/SplittableParDoViaKeyedWorkItems.java, so no loner explicit register SplittableParDoViaKeyedWorkItems?

Reply:
That's right. Without the relocation from the other long comment, this one and the one in the flink runner conflict. But they register the same thing, and it doesn't do anything, and it is totally standard.